SPECS/bind/bind.spec

```diff
@@ -545,9 +541,14 @@ fi;
 %{_mandir}/man8/named-compilezone.8*
 %{_mandir}/man8/named-nzd2nzf.8*
 %{_sysconfdir}/*
-%{_lib}/tmpfiles.d/named.conf
+%{_tmpfilesdir}/named.conf
 
 %changelog
+* Thu May 13 2021 Henry Li <lihl@microsoft.com> - 9.16.3-4
+- Fix file path error caused by linting
+- Remove duplicate %files section for bind-license
```

Comment:
Nit: could you use a double "%" sign? Otherwise the tools are complaining that there's a macro in a comment.